### DISCUSSION TITLE
ASAP7 LEF file for IO bump

### DISCUSSION THREAD

**Discussion by lustefan97** [The-OpenROAD-Project/OpenROAD#4115] [Q&A] [answered]
Hello,

I am looking into the utilization of the PAD tool in order to place an IO bump array. I've look into the test scripts that are located in _OpenROAD/src/pad/test/_ to see how to implement IO bumps in a design. 

My question simply has to do with ASAP7 and the bumps LEF files/object :  I see that an IO LEF file containing a DUMMY_BUMP LEF object has been added/created for the Nangate45 PDK as well as for skywater130, but I haven't found anything similar for the ASAP7 PDK (at least in the OpenROAD github repository). 

Does such a file exist somewhere, or does it need to be created ?

**Comment by maliberty** [comment, marked as answer]
I think it would have to be created.
